docker/ubuntu_18.04-dev.dockerfile

```
@@ -98,7 +98,7 @@ RUN if [ "$build_moab" = "YES" ] || [ "$enable_pymoab" = "YES" ] ; then \
         && cd $HOME/opt \
         && mkdir moab \
         && cd moab \
-        && git clone --single-branch -b Version5.1.0 https://bitbucket.org/fathomteam/moab \
+        && git clone --single-branch -b Version5.2.0 https://bitbucket.org/fathomteam/moab \
```

version 5.2.1 is not available as a branch, just a tag
```suggestion
        && git clone --single-branch -b 5.2.1 https://bitbucket.org/fathomteam/moab \
```